### DISCUSSION TITLE
Prevent dns over https

### DISCUSSION THREAD

**Discussion by laurentftech** [AdguardTeam/AdGuardHome#3821] [Q&A]
Hi. 
i am using AGH as parental control at home. 
Google chrome and Firefox have now a built-in DOH capability which allow the user to circumvent  DNS filtering. By adding some DNS rewrite, it seems possible to disable access to the DOH providers. This is not easy though. 
Is their a list somewhere of the DOH servers to block or another solution available to solve this issue? 
Thank you.

**Comment by ghost** [comment]
I use my DoH block list for this purpose. You also need to redirect DNS port 53 to the internal server in the router and block port 853 (DoT).

Blocklist:
https://raw.githubusercontent.com/Zelo72/adguard/main/doh-vpn-proxy-bypass.adblock

Greeting,
Gerd - https://github.com/Zelo72/adguard
